client/sites/logs/hooks/use-fields.tsx

```diff
@@ -88,8 +88,22 @@ const useFields = ( { logType }: { logType: LogType } ): Field< ServerLog | PHPL
 					},
 					enableSorting: false,
 				},
-				{ id: 'kind', type: 'text', label: translate( 'Kind' ), enableSorting: false },
-				{ id: 'name', type: 'text', label: translate( 'Name' ), enableSorting: false },
+				{
+					id: 'kind',
+					type: 'text',
+					label: translate( 'Kind' ),
+					render: ( { item }: { item: PHPLog } ) =>
+						item.kind !== '' ? item.kind : translate( 'core' ),
+					enableSorting: false,
+				},
+				{
+					id: 'name',
+					type: 'text',
+					label: translate( 'Source' ),
+					render: ( { item }: { item: PHPLog } ) =>
+						item.name !== '' ? item.name : translate( 'core' ),
```

Comment:
I went with lowercase here because all other data does the same (this is plugin/theme names).